hermes-guide-days-website/src/app/footer/footer.component.css

```diff
@@ -0,0 +1,134 @@
+* {
+  color: #fff;
+  font-family: "Montserrat", sans-serif;
+}
+
+.footer {
+  background-color: #333;
+  padding-top: 40px;
+  text-align: center;
+}
+
+#contact {
+  margin-top: 0px;
+  margin-bottom: 40px;
+  font-size: 30px;
+  padding: 0;
+}
+
+.title {
+  font-size: 24px;
+}
+
+.content {
+  font-size: 20px;
+  margin-bottom: 40px;
+}
+
+.col-1-3 {
+  vertical-align: top;
+  display: inline-block;
+  margin-top: 0%;
```

Comment:
since you use all margins use just margin: with 4 values, margin: 4% 0% 0% 0% - make sure the order is ok :D